modules/ols-providing-custom-knowledge-to-the-llm.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="providing-custom-knowledge-to-the-llm_{context}"]
+= Providing custom knowledge to the LLM
+
+Customize the information available to the large language model (LLM) by providing access to a container image that resides in a remote image registry. The examples in this procedure use `quay.io` as the remote container image registry, and the path for the custom image is `quay.io/<username>/my-byok-image:latest`.
+
+:FeatureName: The BYO Knowledge feature
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role. Alternatively, you are logged in to a user account that has permission to create a cluster-scoped CR file.
```

Comment:
idk about providing alternatives here per se. To me, I think you can just say:
```suggestion
* You are logged in to the {ocp-product-title} web console as a user account that has permission to create a cluster-scoped custom resource (CR) file.
```
or, if it's important/helpful to point out that the cluster admin has those perms, something like this?
```suggestion
* You are logged in to the {ocp-product-title} web console as a user account that has permission to create a cluster-scoped custom resource (CR) file, such as a user with the `cluster-admin` role.
```